scripts/mobility-database-harvester/harvest_latest_versions.py

```diff
@@ -11,79 +11,28 @@
 # limitations under the License.
 
 import argparse
-
+import json
+import pandas as pd
 import numpy as np
-import requests
-from datetime import datetime as dt
 import os
 from os import path
-import json
-from tqdm import tqdm
 
-###############################################################################
-# This script harvests the latest dataset versions on the Mobility Database.
+#####################################################################################
+# This script harvests the latest dataset versions on the Mobility Database Catalogs.
 # Made for Python 3.9. Requires the modules listed in requirements.txt.
-###############################################################################
-
-# API constants
-API_URL = "http://old.mobilitydatabase.org/w/api.php?"
-ACTION = "action"
-ARCHIVES_ID_PROPERTY = "P33"
-CLAIMS = "claims"
-ENTITIES = "entities"
-IDS = "ids"
-MODIFIED = "modified"
-SOURCE_PROPERTY = "P5"
-WBGETENTITIES = "wbgetentities"
-FORMAT = "format"
-JSON = "json"
-MAINSNAK = "mainsnak"
-DATAVALUE = "datavalue"
-VALUE = "value"
-ID = "id"
-TYPE = "TYPE"
-PROJECT_ID = "PROJECT_ID"
-PRIVATE_KEY_ID = "PRIVATE_KEY_ID"
-PRIVATE_KEY = "PRIVATE_KEY"
-CLIENT_EMAIL = "CLIENT_EMAIL"
-CLIENT_ID = "CLIENT_ID"
-AUTH_URI = "AUTH_URI"
-TOKEN_URI = "TOKEN_URI"
-AUTH_PROVIDER_X509_CERT_URL = "AUTH_PROVIDER_X509_CERT_URL"
-CLIENT_X509_CERT_URL = "CLIENT_X509_CERT_URL"
-DATASET_PROPERTY = "P15"
-DOWNLOAD_DATE = "P32"
-URL_PROPERTY = "P13"
-
-# Credentials keys
-TYPE_KEY = "type"
-PROJECT_ID_KEY = "project_id"
-PRIVATE_KEY_ID_KEY = "private_key_id"
-PRIVATE_KEY_KEY = "private_key"
-CLIENT_EMAIL_KEY = "client_email"
-CLIENT_ID_KEY = "client_id"
-AUTH_URI_KEY = "auth_uri"
-TOKEN_URI_KEY = "token_uri"
-AUTH_PROVIDER_X509_CERT_URL_KEY = "auth_provider_x509_cert_url"
-CLIENT_X509_CERT_URL_KEY = "client_x509_cert_url"
+#####################################################################################
 
-# Catalog constants
-GTFS_CATALOG_ID = "Q6"
-
-# Archives ids file constants
-ARCHIVES_IDS = "archives_ids"
-HARVESTING_DATE = "harvesting_date"
-EPOCH_DATE = "1970-01-01T00:00:00Z"
+# Mobility Database Catalogs constants
+CATALOGS_CSV = "https://bit.ly/catalogs-csv"
```

Comment:
I'll admit I was a little nervous seeing a bit.ly link here, since I wasn't totally sure what it pointed to.  Given that you are using the expanded url below for URL_PREFIX, would it be reasonable to use the expanded url here as well?

Reply:
Sure.